### DISCUSSION TITLE
Notify user on new comments

### DISCUSSION THREAD

**Discussion by nalafod** [nocodb/nocodb#6228] [Features]
As we are using a lot nocodb i would like to make a suggestion for a new feature that maybe could be implemented on the road map. 

When a user makes a new comment and maybe mention another user in a line with @ user. Then a notification to be received on the mentioned user so to be aware a new comments or discussion is created.

**Comment by nalafod** [comment]
Just to add here i am adding this because there is a mail function to send user password reminder built inside

**Comment by bambus1980** [comment]
Nice suggestion. But I think it will be better to add new functional - trigger list, where would be comments triggers too.

**Reply by nalafod** [reply]
Yes trigger also option would be great

**Comment by bmgtr** [comment]
+1

Additional suggestion: Additionally change background color of a record if _any_ (customizable) field changed, not just comments.

We evaluate nocodb as a bug tracker and this would be extremly helpful to easily see input from the team.
